src/dvclive/live.py

```diff
@@ -183,6 +175,22 @@ def _init_dvc(self):
             mark_dvclive_only_started(self._exp_name)
             self._include_untracked.append(self.dir)
 
+    def _init_dvc_pipeline(self):
+        if os.getenv(env.DVC_EXP_BASELINE_REV, None):
+            # `dvc exp` execution
+            self._baseline_rev = os.getenv(env.DVC_EXP_BASELINE_REV, "")
+            self._exp_name = os.getenv(env.DVC_EXP_NAME, "")
+            self._inside_dvc_exp = True
+            if self._save_dvc_exp:
+                logger.info("Ignoring `save_dvc_exp` because `dvc exp run` is running")
+                self._save_dvc_exp = False
+        elif os.getenv(env.DVC_ROOT, None):
+            # `dvc repro` execution
```

Comment:
@dberenbaum we would also want to ignore `save_dvc_exp` here, right? Like above:

```py
            if self._save_dvc_exp:
                logger.info("Ignoring `save_dvc_exp` because `dvc exp run` is running")
                self._save_dvc_exp = False
```

Reply:
@daavoo Are you aware of any other features that don't work with `dvc repro`? Do we need a warning if `save_dvc_exp=False`?